app/Http/Requests/ResourceItem/UpdateResourceItemRequest.php

```
@@ -27,6 +28,12 @@ public function rules(): array
     {
         return [
             'task_id' => 'required|exists:tasks,id',
+            'setup_item_profile_id' => [
+                'nullable',
+                Rule::requiredIf(function () {
+                    return $this->status === ResourceStatus::ITEM->value && $this->resource_type === ResourceType::MATERIALS->value;
+                })
```

PHPMD (2.15.0)

Avoid using static access to class '\Illuminate\Validation\Rule' in method 'rules'. (undefined)

(StaticAccess)